platforms/hdp2.6.3/docs/HDP-installation.md

```diff
@@ -98,6 +99,17 @@ installed.Then click next.
 
 <img src="docs/dependent_config.jpeg" alt="dependent_config.jpeg" width="400" height="200">
 
+**Note**: Ambari will not show all the recommend values in some situations. Make sure these important configurations are set properly by click "recommend" button next to the configs:
```

Comment:
Change to: "**Note**: Ambari will not show all the recommended values in some situations. Make sure these important configurations are set properly by clicking the "recommend" button next to the configs:"